### ISSUE TITLE
Parse JSON output instead of human readable output in order to fix truncated inspections

### ISSUE THREAD

**Issue by james-johnston-thumbtack** [anboralabs/sqlfluff-intellij-community#22]
The plugin currently parses the default human readable output that is outputted by SQLFluff:

https://github.com/anboralabs/sqlfluff-intellij-community/blob/a5232e374ea50e7f91f5006440628de726d8a07e/src/main/kotlin/co/anbora/labs/sqlfluff/lint/Linter.kt#L73

The problem with this is that it lends itself to parsing mistakes, since the format isn't designed for other programs to parse.  This is especially a problem with multiline violation output:

```
% sqlfluff lint . | cat
== [tests/unit/testdata/databases/nesteddb/20230315_nested_up.sql] FAIL
L:   1 | P:   1 |  TMP | Unrecoverable failure in Jinja templating: A migration
                       | was nested inside of another migration. Have you
                       | configured your variables?
                       | https://docs.sqlfluff.com/en/latest/configuration.html
```

Only the first line is parsed by the plugin as:

<img width="846" alt="Screenshot 2023-03-31 at 9 28 43 PM" src="https://user-images.githubusercontent.com/22308682/229265611-96027d04-569a-4c5f-aed4-6eb8ca7f7ae2.png">

Potentially the `parseLintResult` function could be improved to read multiline errors.  But I think a better solution would be a completely different approach to parse JSON output instead using the `--format json` flag: https://docs.sqlfluff.com/en/stable/cli.html#cmdoption-sqlfluff-lint-f

> What format to return the lint result in (default=human).
> 
> Options:
> human | json | yaml | github-annotation | github-annotation-native

If you set json output, you get something like this:

```json
[
  {
    "filepath": "tests/unit/testdata/databases/nesteddb/20230315_nested_up.sql",
    "violations": [
      {
        "line_no": 1,
        "line_pos": 1,
        "code": "TMP",
        "description": "Unrecoverable failure in Jinja templating: A migration was nested inside of another migration. Have you configured your variables? https://docs.sqlfluff.com/en/latest/configuration.html",
        "name": ""
      }
    ]
  }
]
```

That would be much more readily usable by a plugin and wouldn't have the original problem.

**Comment by dalgarins** [comment]
@james-johnston-thumbtack thanks it is a great approach, I will be working on that.

**Comment by dalgarins** [comment]
fix will be available on version: 1.3.5

**Comment by james-johnston-thumbtack** [comment]
@dalgarins installed the new version today.  Works great!  Thanks for the quick update & new version 😄
